python/cudf/tests/test_binops.py

```diff
@@ -416,9 +416,15 @@ def test_different_shapes_and_columns(binop):
 
 @pytest.mark.parametrize('binop', _binops)
 def test_different_shapes_and_same_columns(binop):
-    import cudf
-    with pytest.raises(NotImplementedError):
-        binop(cudf.DataFrame({'x': [1, 2]}), cudf.DataFrame({'x': [1, 2, 3]}))
+    if binop is operator.pow:
+        return
```

Comment:
Is `operator.pow` not implemented yet? If so we should xfail it instead of just returning True.

Reply:
I'm not sure, but I used the implementation of the test above this as a guide, which has this check:
https://github.com/rapidsai/cudf/blob/62e478d82031acad0b16aff336c02c056f306333/python/cudf/tests/test_binops.py#L386-L389

Reply:
👍